lib/index.js

```diff
@@ -252,18 +252,22 @@ exports.DOMString = (value, options = {}) => {
 
 exports.ByteString = (value, options = {}) => {
   const x = exports.DOMString(value, options);
-  let c;
-  for (let i = 0; (c = x.codePointAt(i)) !== undefined; ++i) {
-    if (c > 255) {
-      throw makeException(TypeError, "is not a valid ByteString", options);
-    }
+
+  // eslint-disable-next-line require-unicode-regexp
+  if (!/[^\x00-\xff]/.test(x)) {
+    return x;
   }
 
-  return x;
+  throw makeException(TypeError, "is not a valid ByteString", options);
 };
 
 exports.USVString = (value, options = {}) => {
```

Comment:
Can't we just replace this whole function with `toWellFormed()`?

Reply:
Done! (When present)